lightningd/log.c

```diff
@@ -564,40 +564,66 @@ void logv(struct logger *log, enum log_level level,
 	  const char *fmt, va_list ap)
 {
 	int save_errno = errno;
-	struct log_entry *l = new_log_entry(log, level, node_id);
 	char *log_msg = NULL;
 
 	/* This is WARN_UNUSED_RESULT, because everyone should somehow deal
 	 * with OOM, even though nobody does. */
 	if (vasprintf(&log_msg, fmt, ap) == -1)
 		abort();
 
-	char **lines = tal_strsplit(
-	    log, json_escape_unescape(log, (struct json_escape *)log_msg), "\n",
-	    STR_NO_EMPTY);
```

Comment:
STR_EMPTY_OK, since you want to log empty lines too I think?

Reply:
Now the output will look something like:
```sh
 $ bitcoin-cli -rpcclienttimeout=60 echo 'hello world'
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: 
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: Could not connect to bitcoind using bitcoin-cli. Is bitcoind running?
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: 
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: Make sure you have bitcoind running and that bitcoin-cli is able to connect to bitcoind.
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: 
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: You can verify that your Bitcoin Core installation is ready for use by running:
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli: 
2024-12-19T06:39:55.230Z **BROKEN** plugin-bcli:     $ bitcoin-cli -rpcclienttimeout=60 echo 'hello world'
2024-12-19T06:39:55.230Z INFO    plugin-bcli: Killing plugin: exited before we sent init
The Bitcoin backend died.
```

Reply:
I think that's right, TBH?